src/state/zustand/swapperStore/amountSelectors.ts

```diff
@@ -199,8 +199,39 @@ export const selectBuyAssetTradeFeeBuyAssetBaseUnit = createSelector(
   },
 )
 
+export const selectProtocolFees = createDeepEqualOutputSelector(
```

Comment:
This is handy - we'll be able to update this selector as we move to the new fee array, and the consumers will be non-the-wiser.